.github/workflows/minecraft.yml

```diff
@@ -0,0 +1,37 @@
+name: Compile TASmod-OG
+
+on:
+  push:
+    branches: [ "develop" ]
+
+jobs:
+  tasmodog:
+    permissions: write-all
+    runs-on: windows-latest
+    steps:
```

Comment:
Add an `if: github.repository == 'MinecraftTAS/TASmod-OG'`, so the fork of the project won't fail the build since it doesn't have the webhook...